examples/allennlp/allennlp_simple.py

```diff
@@ -123,7 +118,7 @@ def objective(trial):
         validation_data_loader=validation_data_loader,
         validation_metric="+" + TARGET_METRIC,
         patience=None,  # `patience=None` since it could conflict with AllenNLPPruningCallback
-        num_epochs=50,
+        num_epochs=30,
```

Comment:
I decreased the number of epochs for reducing the execution time for each trial.